### PR TITLE
#81 added tests and fixed polish translation

### LINKED ISSUE #81: Minimize with main button not possible
It is not possible anymore to minimize the UI by clicking on the main button. This worked before.

### PR DESCRIPTION
**Pull request by JonasJurczok**
fix #81

### CODE REVIEW

**Comment by JonasJurczok** [comment]
preliminary. The actual fix is still missing.

File: README.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Credit where credit is due :)
 ## Changelog
 
 ### NEXT
+* Fixed a bug that the main button does not hide the UI anymore (#81)
```
Comment (Tarrke) [inline README.md]: If this does what I think it does, that's great!

File: src/todo/features/main_ui.lua [hunk 2]
```diff
@@ -87,11 +91,10 @@ function todo.update_current_task_label(player)
     for _, task in pairs(global.todo.open) do
         if task.assignee == player.name then
             todo.log(serpent.block(task))
-            todo.get_maximize_button(player).caption =
-            {"",
-             {"todo.todo_list"},
-             ": ",
-             task.title
+            todo.get_maximize_button(player).caption = { "",
```
Comment (Tarrke) [inline src/todo/features/main_ui.lua]: I think this is not really readable. Out of the box I would have done something like this:
```
todo.get_maximize_button(player).caption = {
    "", { "todo.todo_list" }, ": ", task.title
}
```

Comment (JonasJurczok) [reply src/todo/features/main_ui.lua]: Then the line is longer than the recommended length from luacheck :)
But yes.. it's not optimal.

**Pull request merged**